app/models/campaign.rb

```diff
@@ -18,6 +18,7 @@
 #= Campaign model
 class Campaign < ActiveRecord::Base
   has_many :campaigns_courses, class_name: CampaignsCourses
+  has_many :campaigns_users, class_name: CampaignsUsers, dependent: :destroy
```

Comment:
User needs the same `has_many` (without the dependent bit, as we don't destroy users)